Libraries/CustomComponents/NavigationExperimental/NavigationCard.js

```diff
@@ -87,6 +87,8 @@ class NavigationCard extends React.Component<any, Props, any> {
       ...props,
     } = this.props;
 
+    const defaultStyle = NavigationCardStackStyleInterpolator.forHorizontal(props);
```

Comment:
I debated the same thing about the back button, and ended up doing it for completeness but would be fine excluding such a change.

As for the necessity of `defaultStyle`, requiring the dev to dig into the source to know that `NavigationCardStackStyleInterpolator` is required to be part of it in order for the animation to work is kind of a high bar, and will be the source of many questions as to "why is my push/pop not animating?" Combining the two at the lower level will lead to less questions, and it can still be overridden as written.

However, I did realize that the `if (style === undefined)` is unnecessary, so I'm going to take that out.